### PR TITLE
[Storage] Support VLW and account level worm GA

### PR DESCRIPTION
**Pull request by blueww**
## Description

VLW : The design pass review: Azure/azure-powershell-cmdlet-review-pr#959
Account level worm: design pass review : Azure/azure-powershell-cmdlet-review-pr#1026

Both feature already in Storage preview module.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/Storage.Management/Az.Storage.psd1 [hunk 1]
```diff
@@ -192,7 +192,9 @@ CmdletsToExport = 'Get-AzStorageAccount', 'Get-AzStorageAccountKey',
                'Set-AzStorageBlobTag', 'Get-AzStorageBlobTag', 
                'Get-AzStorageBlobByTag', 
                'Invoke-AzStorageAccountHierarchicalNamespaceUpgrade', 
-               'Stop-AzStorageAccountHierarchicalNamespaceUpgrade'
+               'Stop-AzStorageAccountHierarchicalNamespaceUpgrade', 
+               'Set-AzStorageBlobImmutabilityPolicy', 'Remove-AzStorageBlobImmutabilityPolicy',
+               'Set-AzStorageBlobLegalHold', 'Invoke-AzRmStorageContainerImmutableStorageWithVersioningMigration'
```
Comment (isra-fel) [inline src/Storage/Storage.Management/Az.Storage.psd1:197]: Is there any way we can shorten the name by using abbreviations as alias?

File: src/Storage/Storage.Management/Models/PSContainer.cs [hunk 2]
```diff
@@ -48,6 +48,7 @@ public PSContainer(StorageModels.ListContainerItem container)
             this.DefaultEncryptionScope = container.DefaultEncryptionScope;
             this.DenyEncryptionScopeOverride = container.DenyEncryptionScopeOverride;
             this.Deleted = container.Deleted;
+            this.ImmutableStorageWithVersioning = container.ImmutableStorageWithVersioning is null ? null : new PSImmutableStorageWithVersioning(container.ImmutableStorageWithVersioning);
```
Comment (isra-fel) [inline src/Storage/Storage.Management/Models/PSContainer.cs:51]: This is equivalant to
```suggestion
            this.ImmutableStorageWithVersioning = container.ImmutableStorageWithVersioning ?? new PSImmutableStorageWithVersioning(container.ImmutableStorageWithVersioning);
```

**Review by isra-fel — COMMENTED** [review]
Genrally LGTM

File: src/Storage/Storage.Management/Az.Storage.psd1 (hunk 1 above)
Comment (blueww) [reply src/Storage/Storage.Management/Az.Storage.psd1:197]: We have discussed with PM for that, and finally get not short it.

File: src/Storage/Storage.Management/Models/PSContainer.cs (hunk 2 above)
Comment (blueww) [reply src/Storage/Storage.Management/Models/PSContainer.cs]: fixed

Comment (isra-fel) [reply src/Storage/Storage.Management/Models/PSContainer.cs]: My bad - I can't use `??` between different types. Please revert the last commit. Sorry for the inconvenience!

Comment (blueww) [reply src/Storage/Storage.Management/Models/PSContainer.cs:51]: reverted

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**